### ISSUE TITLE
allow integer input

### ISSUE THREAD

**Issue by Jeffrey-Sarnoff** [dpsanders/CRlibm.jl#8]
julia> exp(-2,RoundNearest)
ERROR: MethodError: `exp` has no method matching exp(::Int64, ::RoundingMode{:Nearest})
Closest candidates are:
  exp(::BigFloat, ::RoundingMode{:Nearest})
  exp(::Float64, ::RoundingMode{:Nearest})
  exp(::Real)
 in eval at ./boot.jl:267

julia> exp(-2.0,RoundNearest)
0.1353352832366127

**Comment by dpsanders** [comment]
Would it be enough to do
`Base.exp(x::Real, r::RoundingMode) = exp(float(x), r)`?

**Comment by dpsanders** [comment]
I have implemented this in #10, but I can't seem to get rid of the following warning:

``` julia
ARNING: New definition
    log(Real, Base.Rounding.RoundingMode) at /Users/dpsanders/.julia/v0.5/CRlibm/src/CRlibm.jl:142
is ambiguous with:
    log(Base.Irrational{:e}, Any) at irrationals.jl:130.
To fix, define
    log(Base.Irrational{:e}, Base.Rounding.RoundingMode)
before the new definition.
```

I have tried adding

```
log(x::Irrational{:e}, r::RoundingMode) = 1  
```

but no luck. Any ideas?

**Comment by dpsanders** [comment]
Fixed in #10, tagged as 0.2.3. Thanks!
